src/xcoobee/api/ConsentsApi.js

```diff
@@ -40,6 +40,41 @@ const confirmConsentChange = (apiUrlRoot, apiAccessToken, consentCursor) => {
     });
 };
 
+/**
+ * Allows a company to open consent related dispute
+ *
+ * @async
+ * @param {string} apiUrlRoot - The root of the API URL.
+ * @param {ApiAccessToken} apiAccessToken - A valid API access token.
+ * @param {*} consentCursor
+ *
+ * @returns {Promise<Object>} - The result.
+ * @property {boolean} confirmed - Flag indicating whether the change is confirmed.

```

Comment:
Please decide whether you want to return `confirmed` or `declined`. Here in the API layer, we are returning `declined` but its documented as `confirmed`. The unit test is expecting `declined`. In the SDK layer, we are returning `declined` but its documented as `confirmed`. The unit test is expecting `confirmed` due to the mock return value.

I am okay with whether you choose `declined` or `confirmed`. Please make it the same throughout the code base.

Reply:
Sorry, I missed it. I'll change it shortly